composer.json

```diff
@@ -67,7 +67,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "psr/log": "~1.0.2"
+        "psr/log": "~1.1.0"
```

Comment:
Why not `^1.1`? This package follows semver.

Reply:
Fixed, thanks!